generators/app/tasks/gitConfig.js

```diff
@@ -0,0 +1,86 @@
+const runCommand = require('./runCommand');
+
+module.exports = function configGit() {
+  return runCommand({
+    command: ['git', ['init', `./${this.projectName}`]],
+    loadingMessage: 'Initiatialization of Git configuration',
+    successMessage: 'Git configuration initiated successfully',
+    failureMessage: 'Git configuration initialization has failed'
+  })
+    .then(() =>
+      runCommand({
+        command: [
+          'git',
+          ['remote', 'add', 'origin', this.repoUrl],
+          { cwd: `${process.cwd()}/${this.projectName}` }
+        ],
+        loadingMessage: 'Adding remote origin url',
+        successMessage: 'Remote origin url added successfully',
+        failureMessage: 'Remote origin url addition has failed'
+      })
+    )
+    .then(() =>
+      runCommand({
+        command: ['mv', ['README.md', 'README_TEMP.md'], { cwd: `${process.cwd()}/${this.projectName}` }],
+        loadingMessage: 'Renaming README.md',
+        successMessage: 'README.md renamed successfully',
+        failureMessage: 'Renaming of README.md has failed'
+      })
+    )
+    .then(() =>
+      runCommand({
+        command: ['git', ['pull', 'origin', 'master'], { cwd: `${process.cwd()}/${this.projectName}` }],
+        loadingMessage: 'Git pull master',
+        successMessage: 'Git pull of master successfully',
+        failureMessage: 'Git pull of master has failed'
+      })
+    )
+    .then(() =>
+      runCommand({
+        command: ['rm', ['README.md'], { cwd: `${process.cwd()}/${this.projectName}` }],
+        loadingMessage: 'Removing default README.md',
+        successMessage: 'Default README.md removed successfully',
+        failureMessage: 'Removing of default README.md has failed'
```

Comment:
Be aware that if readme.md cant be deleted by permissions, then the rest of the commands wont be executed. Maybe you could split the readme actions and the git actions in two different "promise chain" (i mean two different consecutive "then"s chains). 
Finishing the git configuration is far more important than using the repo url in the readme, and doing this last action may bring some permission trouble, based on the folder configuration.